src/board/Cell.css

```diff
@@ -2,15 +2,24 @@
   display: flex;
   align-items: center;
   justify-content: center;
-  background-color: lightpink;
+  background-color: pink;
   border: 1px solid darkred;
   padding: 0;
   font-size: 50px;
   font-weight: bold;
-  color: darkred;
+  color: white;
+  text-shadow: 2px 2px 2px rgba(0, 0, 0, 0.2);
+
   aspect-ratio: 1/1;
   border-radius: 10px;
   box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1),
     inset 2px 2px 4px rgba(255, 255, 255, 0.3),
     inset -2px -2px 4px rgba(0, 0, 0, 0.1);
+  transition: transform 0.9s;
+}
+
+.cell.flipped {
+  transform: rotateY(360deg);
+  color: black;
+  background-color: rgb(255, 255, 255);
 }
```

Comment:
Wow you really got into the complex css stuff; pretty cool. You have just out-learned css compared to me!